benchopt/base.py

```diff
@@ -443,6 +443,30 @@ def save_final_results(self, **solver_result):
         """
         pass
 
+    def format_objective_dict(self, objective_dict):
+        """Format that the output of the objective.
```

Comment:
```suggestion
        """Format the output of the objective.
        
        This will prefix all keys in the dictionary with `objective_`
        to make the objective part of the results clear.
```